src/parser/core/modules/Downtime.js

```diff
@@ -20,19 +20,35 @@ export default class Downtime extends Module {
 		}
 
 		// Merge the downtimes that overlap
-		const finalDowntime = [downtimePeriods.shift()]
+		const finalDowntimes = [downtimePeriods.shift()]
 		downtimePeriods.forEach(dt => {
-			const last = finalDowntime[finalDowntime.length - 1]
+			const last = finalDowntimes[finalDowntimes.length - 1]
 			if (dt.start <= last.end) {
 				if (dt.end > last.end) {
 					last.end = dt.end
 				}
 			} else {
-				finalDowntime.push(dt)
+				finalDowntimes.push(dt)
 			}
 		})
 
+		return finalDowntimes
+	}
+
+	// -----
+	// Style Precedent:
+	// accumulator for downtime function must continue to be called "uptime"
+	// -----
+	getDowntime(start = 0, end = this.parser.currentTimeStamp) {
 		// Return the final number
-		return finalDowntime.reduce((uptime, invuln) => uptime + Math.min(invuln.end, end) - Math.max(invuln.start, start), 0)
+		return this._internalDowntime(start, end).reduce((uptime, invuln) => uptime + Math.min(invuln.end, end) - Math.max(invuln.start, start), 0)
+	}
+
+	getDowntimeWindows(start = 0, end = this.parser.currentTimestamp, minimumWindowSize = -1) {
```

Comment:
From naming convention I would assume this would give me start/end times.
Maybe consider adding a function returning those with this name, renaming this to `getDowntimes` instead, since they are just a list of durations, from what I can tell and we already have a function called `getDowntime`, that gives us the duration.